javascript-methods/main.js

```diff
@@ -0,0 +1,58 @@
+var num1 = 5;
+var num2 = 15;
+var num3 = 7;
+
+var maximumValue = Math.max(num1, num2, num3);
+
+console.log('value of maximumValue:', maximumValue);
+
+var heroes = ['spiderman', 'baymax', 'mystique', 'the incredibles'];
+
+var randomNumber = Math.random();
+randomNumber = randomNumber * heroes.length;
+
+var randomIndex = Math.floor(randomNumber);
+
+console.log('the value of randomIndex:', randomIndex);
+
+var randomHero = heroes[randomIndex];
+
+console.log('value of randomHero:', randomHero);
+
+var library = [{ title: 'Jade City', author: 'Fonda Lee' }, { title: 'Death On The Nile', Author: 'Agatha Christie' }, { title: 'The Metamorphosis', Author: 'Franz Kafka' }];
+
+var lastBook = library.pop();
+
+console.log('value of lastBook:', lastBook);
+
+var firstBook = library.shift();
+
+console.log('value of firstBook:', firstBook);
+
+var js = {
+  title: 'JavaScript for Impatient Programmers',
+  author: 'Dr. Axel Rauschmayer'
+};
+
+var css = {
+  title: 'CSS Secrets',
+  author: 'Lea Verou'
+};
+
+library.push(js.title);
+library.unshift(css.title);
```

Comment:
Please make sure whole books are being added to `library`, not just their titles:

---

![image](https://user-images.githubusercontent.com/16143771/187736245-edcd26d9-1f87-433c-85c3-01e03ad7ef90.png)